### PR TITLE
fixed errors in ubuntu20.04

### PR DESCRIPTION
**Pull request by wewang3**
This PR is to fix compile error in ubuntu20.04

### CODE REVIEW

File: src/dcap_provider.cpp [hunk 1]
```diff
@@ -234,15 +234,15 @@ bool get_cache_expiration_time(const string &cache_control, const string &url, t
                 return false;
             }
         }
-        catch (std::invalid_argument e)
+        catch (std::invalid_argument& e)
```
Comment (olkroshk) [inline src/dcap_provider.cpp:237]: [nit]:  `std::invalid_argument&` -> `const std::invalid_argument&`

Comment (wewang3) [reply src/dcap_provider.cpp]: changed to const reference. It is a better practice

File: src/Linux/README.MD [hunk 2]
```diff
@@ -73,6 +73,19 @@ build-essential package (`sudo apt install build-essential`) and dh-virtualenv
 
 The output deb file is suitable for installation on any amd64 Ubuntu system.
 
+In Ubuntu 20.04, please try this command to install dh-virtualenv:
```
Comment (deschuma) [inline src/Linux/README.MD:76]: Why are these required? (dh-virtualenv and fakeroot)?
dh-virtualenv looks like it is for python
Can fakeroot be avoided if the user has root privlege?

I'm not a familiar with either of these tools, so please let me know if there are new requirements specific to Ubuntu 20.04 which are requiring them.

File: src/Linux/debian/changelog [hunk 3]
```diff
@@ -1,3 +1,8 @@
+az-dcap-client (1.8) stable; urgency=medium
+  * Enable Ubuntu 20.04 package and Fixed compile errors
```
Comment (deschuma) [inline src/Linux/debian/changelog:2]: Nit: spelling: comile->compile

**Review by deschuma — CHANGES_REQUESTED** [review]
Code looks good. Just left a few comments about the readme and release notes

**Review by deschuma — APPROVED** [review]

**Review by olkroshk — APPROVED** [review]
lgtm, code part looks good to me, I don't have a context on the documentation.
